### PR TITLE
Add RMT automatic testing code for import and export

### PR DESCRIPTION
**Pull request by tinawang123**
Add RMT automatic testing code for import and export

- Related ticket:https://progress.opensuse.org/issues/60374
- Verification run: 
import: https://openqa.suse.de/tests/4474648#
export: https://openqa.suse.de/tests/4474647#

### CODE REVIEW

File: lib/repo_tools.pm [hunk 1]
```diff
@@ -267,23 +275,34 @@ C<$datafile> is repository source.
...
-    assert_script_run("wget -q " . data_url("rmt/$datafile"));
-    assert_script_run("tar -xzvf $datafile -C $datapath");
-    assert_script_run("rm -rf $datafile");
+    my ($datapath) = @_;
+    # Check import data resource exsited
+    assert_script_run("ls $datapath");
     # Import RMT data from test path to new RMT server
     assert_script_run("rmt-cli import data $datapath",  600);
     assert_script_run("rmt-cli import repos $datapath", 600);
-    # Show repo list on new RMT server for later debugging
```
Comment (lemon-suse) [inline lib/repo_tools.pm:280]: After import data, we need verify it by show enabled repo from the original RMT server.

**Comment by lemon-suse** [comment]
In general, if we later will use the two pm to add RMT related test, it is better to change the module name such as RMT_server_orig and RMT_server_sut. WDYT?

File: tests/x11/rmt/rmt_import.pm [hunk 2]
```diff
@@ -0,0 +1,46 @@
+# SUSE's openQA tests
+#
+# Copyright © 2019 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Add rmt configuration test and disconnect RMT test
```
Comment (coolgw) [inline tests/x11/rmt/rmt_import.pm:10]: disconnect RMT test?

File: tests/x11/rmt/rmt_export.pm [hunk 3]
```diff
@@ -0,0 +1,42 @@
+# SUSE's openQA tests
+#
+# Copyright © 2020 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+#
+# Summary: RMT export data
```
Comment (coolgw) [inline tests/x11/rmt/rmt_export.pm:10]: Acting as rmt support server provide data which used by RMT import client? 
I suppose here we need give more detail for setup.

**Comment by tinawang123** [comment]
Verified run: 
export: https://openqa.nue.suse.com/tests/4482060#
import: https://openqa.nue.suse.com/tests/4482061

File: lib/repo_tools.pm [hunk 4]
```diff
@@ -229,61 +244,101 @@ sub rmt_wizard {
...
-    assert_script_run("rmt-cli repos list");
-    # Enable repositories as required on new RMT server
-    assert_script_run("rmt-cli repos list | grep Web-Scripting");
```
Comment (lemon-suse) [inline lib/repo_tools.pm:283]: Will you verifiy all products or modules imported from original RMT server?

**Comment by tinawang123** [comment]
Verified:
export https://openqa.suse.de/tests/4522899#
import: https://openqa.suse.de/tests/4522900#step/rmt_import/56

**Comment by tinawang123** [comment]
> In general, if we later will use the two pm to add RMT related test, it is better to change the module name such as RMT_server_orig and RMT_server_sut. WDYT?

As I only test RMT import and export feature. I think use this name can easy to understand what's the file doing. I will keep use rmt_export and rmt_import. Thanks for your suggestion.

**Pull request merged**